prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -135,6 +138,25 @@ function createCluster() {
 	env ADDITIONAL_LABELS="created-at=${CURRENT_TIMESTAMP}" "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/provision-gke-cluster.sh
 }
 
+function waitForInstallationCRD() {
+    shout "Waiting for Installation CRD"
+
+	attempts=5
+    for ((i=1; i<=attempts; i++)); do
+        numberOfLines=$(kubectl get crd | grep -c "installations.installer.kyma-project.io")
```

Comment:
it's really doing the job? shouldn't you check the status of CRD. 

IMO it's working basically because you just added some additional time between CRD creation and installation. And this is why this CRD is visible. To make it real you should check if CRD has correct status and it's really visible in core api server